activation/validator.go

```diff
@@ -34,11 +34,11 @@ func (v *Validator) Validate(minerID signing.PublicKey, nipst *types.NIPST, expe
 		return fmt.Errorf("PoET proof chain invalid: %v", err)
 	}
 
-	if nipst.Space < v.postCfg.SpacePerUnit {
-		return fmt.Errorf("PoST space (%d) is less than a single space unit (%d)", nipst.Space, v.postCfg.SpacePerUnit)
+	if space < v.postCfg.SpacePerUnit {
```

Comment:
Not sure about this. The `space` still needs to be passed into this method, because we need it in order to validate the PoST. The validator has access to the post config, which the ATX DB doesn't, so validating it here is much easier.